### PR TITLE
UpdatedLeadership

### PR DESCRIPTION
**Pull request by ChristianASC22**
Swan and Maeve leadership positions fixed in About section.

### CODE REVIEW

File: app/about/page.tsx [hunk 1]
```diff
@@ -116,7 +116,7 @@ export default function About() {
           />
           <Container className="text-center mb-8">
             <Typography variant="body1" className="text-mb font-semibold">
-              NAN
+              Meave (something)
```
Comment (AdamJablonka) [inline app/about/page.tsx]: You should ask Swan for her last name, instead of leaving it as (something), you don't want to push (something) to the website

**Review by AdamJablonka — CHANGES_REQUESTED** [review]

**Review by ChristianASC22 — COMMENTED** [review]
Maeve's last name was added.

**Review by AdamJablonka — CHANGES_REQUESTED** [review]
Awesome this looks great, while you're here, wanna make things more scalable?

We should create an object, with their names, before the render area of the code file. I'll give you a hint here without the types involved so you can infer.

Something like:
```
BoardMemberNames= {
   president: "Swan Htet",
   vicePresident: "Meave Cheevers",
   secretary: "Anand Persaud",
   treasurer: "Ryan Persad",
}
```

Then do you know how you would replace the names within the typography component with these?
something like `{BoardMemberNames.?}`

This allows it to be easier and more scalable in the future when the next board members are enlisted next semester.

**Review by ChristianASC22 — COMMENTED** [review]
In object form.

**Comment by AdamJablonka** [comment]
Looks great

**Review by AdamJablonka — APPROVED** [review]
LGTM!

**Pull request merged**